plans/completed/remove-ts-node-nestjs/remove-ts-node-from-nestjs-templates.md

```
@@ -0,0 +1,321 @@
+6. Update any "in_progress" task in `./plans/completed/remove-ts-node-nestjs/tasks` to "completed"
+7. Commit changes
+8. Push all changes to the pull request
+9. Change the pull request to ready for review
+10. Set the pull request to auto merge
+
+**Skills to Invoke:** `/git:commit`
+**Metadata:** `{ "plan": "replicated-jumping-ladybug", "type": "task", "skills": ["/coding-philosophy", "/git:commit"] }`
+
+## Task Dependencies
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)